src/sentry/models/group.py

```diff
@@ -803,6 +804,23 @@ def get_recommended_event_for_environments(
             else self.get_latest_event_for_environments([env.name for env in environments])
         )
 
+    def get_suspect_commit(self) -> Commit | None:
```

Comment:
It looks like this only works for suspect commits with git blame, but without integrations we still show suspect commits on the issue details page (see https://docs.sentry.io/product/issues/suspect-commits/#missing-suspect-commits and [this comment](https://github.com/getsentry/sentry/blob/a297c789d9c2dbf99d74fdb1872eeb078281b268/src/sentry/utils/committers.py#L339-L343)). We may want to have the same logic here? Also note that you can have multiple suspect commits in this case

Reply:
which suspect commit is shown on the issue page in the case of multiple suspect commits?

Reply:
You can see all by clicking "show more", the first one is the one that is shown most prominently